### DISCUSSION TITLE
Update Minigame new

### DISCUSSION THREAD

**Discussion by damoshack54** [masterking32/MasterHamsterKombatBot#276] [General]
Update Minigame new

Zoopolis
Gangs Wars

Thank you very much.

![Zoopolis_Gangs_Wars](https://github.com/user-attachments/assets/c2256c7c-9108-4005-a7de-50c7afd453eb)

**Comment by al-bani** [comment]
where i should input that words?

**Comment by tboy1337** [comment]
Completed with https://github.com/masterking32/MasterHamsterKombatBot/pull/282
